extmod/uasyncio/taskgroup.py

```diff
@@ -0,0 +1,244 @@
+# Adapted with permission from the EdgeDB project.
+# Adapted for MicroPython
+
+
+__all__ = ["TaskGroup"]
+
+from . import core
+from . import event
+from . import task
+
+DEBUG = False
```

Comment:
If you do `_DEBUG = const(False)` then this will be completely removed by the compiler when false.